### PR TITLE
Initial import

### PR DESCRIPTION
**Pull request by bartdag**
Porting staticfiles_helper to its very own project.

### CODE REVIEW

**Comment by bartdag** [comment]
ping

**Comment by bartdag** [comment]
@ginmrt ping!

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+code is deployed in production.
+
+How does it work?
+-----------------
+
+In its simplest form, django-static-url will compute a hash based on the
+current time and insert this hash between the static url prefix (e.g.,
+/static/) and the static file path. The URL will be recomputed every time the
+devserver is reloaded so any changes to static files should be picked up by the
+browser without having to empty of bypass the cache.
```
Comment (ginmrt) [inline README.rst:22]: of = or?

File: README.rst [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+    if settings.DEBUG:
+        urlpatterns += staticfiles_dynamicurlpatterns(settings.STATIC_ROOT_URL)
+
+
+Add this to your nginx config file (production)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+::
+
+    # regex that capture all requests made to
```
Comment (ginmrt) [inline README.rst:105]: captures

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+    # the version across setup.py and the project code, see
+    # https://packaging.python.org/en/latest/single_source_version.html
+    version='0.1.0',
+
+    description="Dynamically generates new static URLs to invalidate "
+    "browsers' cache",
+    long_description=long_description,
+
+    # The project's main homepage.
+    url='https://github.com/resulto-admin/django-celery-fulldbresult',
```
Comment (ginmrt) [inline setup.py:26]: wrong URL

File: tests/test_project/settings.py [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+
+SECRET_KEY = "abcdef123-"
+
+DEBUG = True
+IN_TEST = True
+
+DATABASES = {
+    "default": {
+        "ENGINE": "django.db.backends.postgresql_psycopg2",
+        # XXX This assumes you have the loyalty project setup on your machine
```
Comment (ginmrt) [inline tests/test_project/settings.py:20]: This was copy-pasted but I don't think you needed postgres here? Should probably remove references to loyalty & resulto_dj

File: tests/test_project/settings.py [hunk 5]
```diff
@@ -0,0 +1,202 @@
+import os
+import sys
+
+# Import source code dir
+#   This ensures that we can import `resulto_dj.xyz...` from within the tests/
```
Comment (ginmrt) [inline tests/test_project/settings.py:5]: resulto_dj ref

File: tests/test_project/settings.py [hunk 6]
```diff
@@ -0,0 +1,202 @@
...
+                "django.contrib.auth.context_processors.auth",
+                "django.template.context_processors.debug",
+                "django.template.context_processors.i18n",
+                "django.template.context_processors.media",
+                "django.template.context_processors.static",
+                "django.template.context_processors.tz",
+                "django.contrib.messages.context_processors.messages",
+            ],
+            "globals": {
+                "get_current_date": "resulto_dj.jinja_lib.get_current_date",
```
Comment (ginmrt) [inline tests/test_project/settings.py:78]: resulto_dj ref

File: tests/test_project/settings.py [hunk 7]
```diff
@@ -0,0 +1,202 @@
...
+            "handlers": ["console"],
+            "level": "ERROR",
+            "propagate": False,
+        },
+        "celery": {
+            "handlers": ["console"],
+            "level": "INFO",
+            "propagate": False,
+        },
+        "resulto_dj": {
```
Comment (ginmrt) [inline tests/test_project/settings.py:143]: resulto_dj ref

**Review by ginmrt — APPROVED** [review]
Some typos to fix and small cleanup necessary, after which g2g

**Pull request closed without merging**